### PR TITLE
Create space travellers-hub

### PR DESCRIPTION
**Pull request by zeff96**
In this project, we @zeff96  and @c00p75  have created a space traveler-hub application and added the following to the project

1. Navbar with nav links that navigates to different sections of the application
2. Fetched rocket data from API
3. Rendered fetched rockets on the screen showing rocket image, name, description
4. Reserve and cancel buttons for reservation of rockets and cancellation
5. Fetched missions data from API
6. Rendered fetched mission on the screen showing name, description, and status
7. Joined and cancel join button for the missions
8. Profile section that's shows the list of joined missions and reserved rockets

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,113 @@
...
-Your app is ready to be deployed!
+- Fetching Rockets and Missions from API
+- Reserve rockets and join missions
+- See reserved rockets and joined missions in my profile page.
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Live Demo
 
-### `npm run eject`
+[Live Demo]()
```
Comment (amejid) [inline README.md]: - [ ] The live demo link is broken. I suggest you remove the entire `Live Demo` section if you are not deploying the app. Alternatively, you can deploy the app and provide the link here.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,113 @@
...
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
```
Comment (amejid) [inline README.md:29]: - [ ] Some of the headers here are not available in the README body. It's not navigating to the corresponding README section. I suggest you take the time and work on your README file to make it as professional as possible.

**Review by amejid — COMMENTED** [review]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, that has a broken link inside the README file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by zeff96** [comment]
Hi @amejid, I accidentally clicked on the request review button and our pull request is now pending your review. anyway you may assist, please

**Comment by amejid** [comment]
Hi @zeff96, kindly submit the PR in the dashboard so it can get reviewed.

**Comment by zeff96** [comment]
@amejid, i did that already

**Review by Owusu-Desmond — APPROVED** [review]
Hi @zeff96 and @c00p75 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great team contribution ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zeff96** [comment]
Hi @Owusu-Desmond, Thanks for the time and effort to reviewing our project, appreciated 🤝

**Pull request merged**